srcalc/src/main/java/gov/va/med/srcalc/vista/VistaPatientDao.java

```diff
@@ -30,6 +30,23 @@ public String getDescription()
             return fDescription;
         }
     }
+    
+    /**
+     * This is a list of the only health factors that should be displayed for the users.
+     * This list was created by the VA.
+     */
+    public static final String[] HEALTH_FACTORS_ARRAY = { "ALCOHOL - TREATMENT REFERRAL", "ALCOHOL USE",
```

Comment:
You know, you could easily make this a Set here (`ImmutableSet.of()`) and you don't have to do any transformation in `RpcVistaPatientDao`.